package.json

```diff
@@ -68,14 +73,14 @@
     "ts-jest": "^25.4.0",
     "ts-node": "^8.8.1",
     "tslib": "^1.11.1",
-    "tslint": "^5.11.0",
-    "tslint-stylish": "^2.1.0",
     "typescript": "^3.5.3",
     "webpack-config-utils": "^2.3.1"
   },
   "scripts": {
     "build": "gulp package",
     "test": "gulp tests",
+    "eslint": "gulp eslint",
```

Comment:
This name should be `lint`. In case we change lint tool in future :)

Reply:
OK :)

Reply:
Done.